Fix capitalisation of Heroic in GlobalState.tsx

LenaWil opened the pull request:
I noticed it while translating.



---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

LenaWil commented:
Am I stupid, or should [e2e](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/actions/runs/12528169796/job/34942495477?pr=4218) not fail?

arielj reviewed (changes requested):
I think you need to run `pnpm i18n` so it updates the locale json file

LenaWil commented:
> I think you need to run `pnpm i18n` so it updates the locale json file

```
> heroic@2.15.2 i18n /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher
> i18next --silent

Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__mocks__/@xhmikosr/decompress-unzip.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__mocks__/config.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__mocks__/constants.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__mocks__/electron-store.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__mocks__/electron.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__mocks__/i18next.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__tests__/cache.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__tests__/constants.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__tests__/main_window.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__tests__/progress_bar.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__tests__/skip.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/__tests__/utils.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/anticheat/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/anticheat/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/api/downloadmanager.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/api/helpers.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/api/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/api/library.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/api/menu.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/api/misc.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/api/settings.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/api/wine.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/backend_events.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/cache.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/config.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/constants.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/dialog/__mocks__/dialog.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/dialog/dialog.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/downloadmanager/downloadqueue.ts
Key is not a string literal
Key is not a string literal
Key is not a string literal
Key is not a string literal
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/downloadmanager/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/downloadmanager/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/electron_store.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/game_config.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/images_cache.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/launcher.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/logger/__mocks__/logfile.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/logger/__mocks__/logger.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/logger/__tests__/logfile.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/logger/__tests__/logger.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/logger/__tests__/uploader.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/logger/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/logger/logfile.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/logger/logger.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/logger/uploader.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/main_window.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/main.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/online_monitor.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/preload.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/progress_bar.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/protocol.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/recent_games/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/recent_games/recent_games.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/save_sync.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/schemas.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/shortcuts/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/shortcuts/nonesteamgame/__tests__/nonesteamgame.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/shortcuts/nonesteamgame/constants.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/shortcuts/nonesteamgame/nonesteamgame.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/shortcuts/nonesteamgame/steamhelper.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/shortcuts/shortcuts/shortcuts.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/shortcuts/types.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/shortcuts/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/gog/e2eMock.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/gog/electronStores.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/gog/games.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/gog/library.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/gog/presence.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/gog/redist.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/gog/setup.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/gog/user.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/__tests__/games.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/auth.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/base.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/cleanup.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/egl_sync.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/eos_overlay.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/import.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/info.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/install.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/launch.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/list.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/move.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/status.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/sync_saves.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/commands/uninstall.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/e2eMock.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/electronStores.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/eos_overlay/eos_overlay.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/eos_overlay/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/games.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/library.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/setup.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/thirdParty.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/legendary/user.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/nile/e2eMock.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/nile/electronStores.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/nile/games.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/nile/library.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/nile/setup.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/nile/user.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/sideload/electronStores.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/sideload/games.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/sideload/library.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/storeManagers/storeManagerCommon/games.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/tools/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/tools/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/tray_icon/__tests__/tray_icon.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/tray_icon/tray_icon.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/updater.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/__tests__/compatibility_layers.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/aborthandler/aborthandler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/compatibility_layers.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/filesystem/__tests__/unix.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/filesystem/__tests__/windows.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/filesystem/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/filesystem/unix.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/filesystem/windows.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/graphics/vulkan/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/helperBinaries/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/inet/downloader/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/os/path/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/os/processes/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/gpu/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/gpu/linux.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/gpu/pci_ids.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/gpu/windows.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/heroicVersion.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/memory/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/memory/linux.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/memory/macos.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/memory/windows.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/osInfo/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/osInfo/linux.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/osInfo/windows.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/utils/systeminfo/steamDeck.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/__mocks__/wiki_game_info.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/applegamingwiki/__tests__/utils.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/applegamingwiki/constants.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/applegamingwiki/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/electronStore.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/gamesdb/__tests__/utils.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/gamesdb/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/howlongtobeat/__tests__/utils.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/howlongtobeat/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/pcgamingwiki/__tests__/utils.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/pcgamingwiki/constants.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/pcgamingwiki/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/protondb/__tests__/utils.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/protondb/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/steamdeck/__tests__/utils.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/steamdeck/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/umu/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wiki_game_info/wiki_game_info.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/manager/downloader/__tests__/main/getter.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/manager/downloader/__tests__/utilities/rest.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/manager/downloader/__tests__/utilities/unzip.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/manager/downloader/constants.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/manager/downloader/main.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/manager/downloader/utilities.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/manager/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/manager/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/runtimes/__tests__/runtimes/utils.test.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/runtimes/ipc_handler.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/runtimes/runtimes.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/backend/wine/runtimes/util.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/typedefs/i18next.d.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/typedefs/ipcBridge.d.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/typedefs/steam-shortcut-editor.d.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/typedefs/vite.d.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/types.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/types/electron_store.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/types/epic-graphql.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/types/frontend_messages.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/types/game_manager.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/types/gog.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/types/legendary.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/types/nile.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/types/proxy-types.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/common/utils.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/App.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/ActionIcons/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Anticheat/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/CachedImage/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/CategoryFilter/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/ChangelogModal/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/ControllerHints/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Dialog/components/Dialog.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Dialog/components/DialogContent.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Dialog/components/DialogFooter.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Dialog/components/DialogHeader.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Dialog/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/DialogHandler/components/MessageBoxModal/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/DialogHandler/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/DLCList/DLC/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/DLCList/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/ErrorComponent/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/ExternalLinkDialog/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/FormControl/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Header/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Help/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/InfoBox/index.tsx
Key is not a string literal: text
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/LanguageSelector/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/LibraryFilters/index.tsx
Key is not a string literal
Key is not a string literal
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/LibrarySearchBar/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/LogFileUploadDialog/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/OfflineMessage/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/PathSelectionBox/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/PopoverComponent/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/ProgressDialog/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/SearchBar/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/SelectField/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Sidebar/components/CurrentDownload/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Sidebar/components/HeroicVersion/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Sidebar/components/QuitButton/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Sidebar/components/SidebarItem/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Sidebar/components/SidebarLinks/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Sidebar/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/SmallInfo/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/StoreLogos/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/SvgButton/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/TabPanel/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/TextInputField/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/TextInputWithIconField/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/TextWithProgress/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/ThemeSelector/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/ToggleSwitch/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/TwoColTableInput/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/UninstallModal/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/UpdateComponent/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/WebviewControls/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/WikiGameInfo/components/GameScore/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/WikiGameInfo/components/HowLongToBeat/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/WindowControls/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/WindowControls/RestoreWindow.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Winetricks/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/components/UI/Winetricks/WinetricksSearch/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/electronStores.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/gamepad_layouts/genius.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/gamepad_layouts/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/gamepad_layouts/nintendo.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/gamepad_layouts/ps.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/gamepad_layouts/standard.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/gamepad.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/library.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/onlineMonitor.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/shortcuts.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/helpers/virtualKeyboard.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/hooks/constants.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/hooks/hasAnticheatInfo.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/hooks/hasHelp.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/hooks/hasProgress.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/hooks/hasStatus.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/hooks/useAwaited.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/hooks/useSetting.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/hooks/useSettingsContext.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Accessibility/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/DownloadManager/components/DownloadManagerItem/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/DownloadManager/components/ProgressHeader/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/DownloadManager/DownloadManagerHeader/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/DownloadManager/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GameChangeLog/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GameContext.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/AppleWikiInfo.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/CloudSavesSync.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/CompatibilityInfo.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/Description.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/Developer.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/DotsMenu.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/DownloadSizeInfo.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/GameStatus.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/Genres.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/HLTB.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/InstalledInfo.tsx
Key is not a string literal
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/LaunchOptions.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/MainButton.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/ReleaseDate.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/ReportIssue.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/Requirements.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/Scores.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/components/SettingsButton.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePage/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GamePicture/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GameRequirements/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/GameSubMenu/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/ModifyInstallModal/GOG/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/ModifyInstallModal/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/ModifyInstallModal/Legendary/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Game/TimeContainer/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/AddGameButton/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/CategoriesManager/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/ContextMenu/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/EmptyLibrary/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/GameCard/constants.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/GameCard/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/GamesList/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/InstallModal/DownloadDialog/BranchSelector.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/InstallModal/DownloadDialog/BuildSelector.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/InstallModal/DownloadDialog/DLCDownloadListing.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/InstallModal/DownloadDialog/GameLanguageSelector.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/InstallModal/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/InstallModal/SideloadDialog/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/InstallModal/ThirdPartyDialog/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/InstallModal/WineSelector/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/LibraryHeader/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/components/RecentlyPlayed/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Library/LibraryContext.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Loading/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Login/components/LoginWarning/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Login/components/Runner/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Login/components/SIDLogin/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Login/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AfterLaunchScriptPath.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AllowInstallationBrokenAnticheat.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AlternativeExe.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AltGOGdlBin.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AltLegendaryBin.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AltNileBin.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AutoDXVK.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AutoDXVKNVAPI.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AutoUpdateGames.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/AutoVKD3D.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/BattlEyeRuntime.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/BeforeLaunchScriptPath.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/CheckUpdatesOnStartup.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/ClearCache.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/CrossoverBottle.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/CustomWineProton.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/DefaultInstallPath.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/DefaultSteamPath.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/DisableController.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/DisableLogs.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/DisableUMU.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/DiscordRPC.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/DownloadNoHTTPS.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/EacRuntime.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/EgsSettings.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/EnableDXVKFpsLimit.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/EnableEsync.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/EnableFSR.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/EnableFsync.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/EnableMsync.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/EnvVariablesTable.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/ExperimentalFeatures.tsx
Key is not a string literal
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/GameMode.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/Gamescope.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/HideChangelogOnStartup.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/IgnoreGameUpdates.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/index.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/LauncherArgs.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/LibraryTopSection.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/Mangohud.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/MaxRecentGames.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/MaxWorkers.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/MinimizeOnGameLaunch.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/OfflineMode.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/PlaytimeSync.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/PreferedLanguage.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/PreferSystemLibs.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/ResetHeroic.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/SettingsModal/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/Shortcuts.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/ShowFPS.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/SteamRuntime.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/Tools/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/TraySettings.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/UseDarkTrayIcon.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/UseDGPU.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/UseFramelessWindow.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/WinePrefix.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/WinePrefixesBasePath.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/WineVersionSelector.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/components/WrappersTable.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/AdvancedSettings/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/CategorySettings/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/FooterInfo/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/GamesSettings/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/GeneralSettings/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/LogSettings/components/UploadedLogFilesList/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/LogSettings/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SyncSaves/gog.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SyncSaves/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SyncSaves/legendary.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SystemInfo/cpu.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SystemInfo/gpu.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SystemInfo/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SystemInfo/memory.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SystemInfo/os.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SystemInfo/software.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/sections/SystemInfo/vendorLogo.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/Settings/SettingsContext.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/WebView/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/WineManager/components/WineItem/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/WineManager/index.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/screens/WineManager/state.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/state/ContextProvider.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/state/GlobalState.tsx
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/state/GlobalStateV2.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/state/UploadedLogFiles.ts
Parsing /Users/lena/Documents/Vrije tijd/HeroicGamesLauncher/src/frontend/types.ts
Found translation key already mapped to a map or parent of new key already mapped to a string: notify.uninstalled
Found same keys with different values: box.shortcuts.title
Found same keys with different values: box.select.exe
Found same keys with different values: setting.eosOverlay.updating
[en] translation
Unique keys: 613 (0 are plurals)
Added keys: 0
Restored keys: 0
Removed keys: 0

[en] gamepage
Unique keys: 198 (0 are plurals)
Added keys: 0
Restored keys: 0
Removed keys: 0

[en] login
Unique keys: 15 (0 are plurals)
Added keys: 0
Restored keys: 0
Removed keys: 0

(node:39140) [DEP0180] DeprecationWarning: fs.Stats constructor is deprecated.
(Use `node --trace-deprecation ...` to show where the warning was created)

```

I did, but `git status` afterwards isn’t reporting anything to push.

LenaWil commented:
@arielj Can you review again?

arielj reviewed (approved):
Looks good, thanks!
